### DISCUSSION TITLE
JBL page 309P Mark ii has a photo of an LSR model

### DISCUSSION THREAD

**Discussion by KipRM** [pierreaubert/spinorama#270] [General]
I notice the photo of the while speaker for a JBL 308P Mark ii is of an LSR model
Center white speaker must be an LSR
![image](https://github.com/pierreaubert/spinorama/assets/18583915/c2d3c337-cf3c-46ca-a8aa-68a0db265abc)

**Comment by pierreaubert** [comment]
thanks. It took me some times to see the differences.
Fixed in `#8b63f3b71`
